ingestr/src/utility.py

```diff
@@ -0,0 +1,18 @@
+import requests
+from dlt.sources.helpers.requests import Client
+
+
+def create_client() -> requests.Session:
```

Comment:
Love it that you've refactored a common piece of code to a standalone module.

It would be great if you could refactor other sources that duplicate this client. You may even want to parameterise this function so users of this package can pass in additional configuration to override the defaults.

Reply:
agreed, although I suggest doing this in another PR